src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -2651,7 +2651,7 @@ def test_managedappdef_inline(self, resource_group):
         curr_dir = os.path.dirname(os.path.realpath(__file__))
 
         self.kwargs.update({
-            'upn': self.create_random_name('testuser', 15) + '@azuresdkteam.onmicrosoft.com',
+            'upn': self.create_random_name('testuser', 15) + '@detienne001gmail.onmicrosoft.com',
```

Comment:
Same as above

Reply:
I do not have the permissions, I had to use my personal account and change the domain to be able to record this test. If you can please record.